src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+
+  display: flex;
+  align-items: center;
+
+  height: 60px;
+
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
You should use a fallback font in case 'Roboto' doesn't load. It's always a good idea to specify a generic fallback font in case the web-based font doesn't load. For example, you could use 'sans-serif' as a fallback. You can do this by adding it after 'Roboto', separated by a comma like so: 'Roboto', sans-serif.